test_scripts/Defects/5_0/1909_SDL_must_set_LIMITED_HMILevel_in_case_app_was_activated_during_postponed_HMILevel_resumption.lua

```diff
@@ -0,0 +1,88 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/1909
+--
+-- Description:
+-- SDL must set LIMITED HMILevel in case app was activated during postponed HMILevel resumption
+-- Precondition:
```

Comment:
Please remove `Precondition`, it is redundant for script, describe all needed steps in section `In case`